sourcegraph/about-replatform
# Port/about pg

This closes #66. It ports over the About page, its images, and the LeadershipSection and BoardSection components. It also transfers styling to Bootstrap utility classes.

### To Test
- Navigate to `/about` to check that the design behaves as expected
- Test the static build
